File: VOLUME 2/ANTIPORNOGRAFIA/ANTIPORNO.js
const TelegramBot = require('node-telegram-bot-api');
const token = 'seu_token_aqui';
const bot = new TelegramBot(token, {polling: true});

bot.on('message', (msg) => {
  // Verifica se a mensagem vem de um grupo
  if (msg.chat.type === 'group') {
    // Verifica se a mensagem contém pornografia
    if (msg.photo || msg.video || msg.animation || msg.sticker) {
      // Remove a mensagem
      bot.deleteMessage(msg.chat.id, msg.message_id);
      // Bane o membro
      bot.kickChatMember(msg.chat.id, msg.from.id);
    }
  }
});

File: MODULO 2/ANTIPORNOGRAFIA/ANTIPORNO.py
import logging
import pyporn
from telegram.ext import Updater, MessageHandler, Filters

# Configuração do bot
TOKEN = 'TOKEN_AQUI'
updater = Updater(TOKEN, use_context=True)
dispatcher = updater.dispatcher

# Configuração do logger
logging.basicConfig(format='%(asctime)s - %(name)s - %(levelname)s - %(message)s',
                     level=logging.INFO)

# Função para lidar com as mensagens
def handle_message(update, context):
    # Obtém a mensagem e o chat em que ela foi enviada
    message = update.message
    chat = message.chat

    # Verifica se a mensagem contém pornografia
    if pyporn.contains_porn(message.text):
        # Remove a mensagem
        context.bot.delete_message(chat.id, message.message_id)
        # Bane o usuário
        context.bot.kick_chat_member(chat.id, message.from_user.id)

# Registra o manipulador de mensagens
dispatcher.add_handler(MessageHandler(Filters.text & Filters.group, handle_message))

# Inicia o bot
updater.start_polling()

